gnome/gnome-radio/Portfile

```diff
@@ -7,6 +7,7 @@ PortGroup           app 1.0
 
 name                gnome-radio
 version             64.0
+replaced_by         gtk-internet-radio-locator
```

Comment:
This is not sufficient for removing a port; you will want to follow the entire process listed in: https://guide.macports.org/#development.practices.rename-replace-port